tslib/src/services/data/check-window.data.service.ts

```diff
@@ -0,0 +1,23 @@
+import { SqlService } from '../../sql/sql.service'
+import { ICheckWindow } from '../../schemas/check-window'
+
+export interface ICheckWindowDataService {
+  getActiveCheckWindow (): Promise<ICheckWindow>
+}
+
+export class CheckWindowDataService implements ICheckWindowDataService {
+
+  private _sqlService: SqlService
```

Comment:
I am not sure there is any reason to prefix a private var with underscore?

Reply:
done